src/js/actions/policy.js

```diff
@@ -213,62 +210,80 @@ define(function (require, exports) {
     removePointerPolicies.writes = [locks.PS_APP, locks.JS_POLICY];
 
     /**
-     * Temporarily disable keyboard policies, caching the previous state
+     * Temporarily disable all policies, caching the previous state
      *
      * @return {Promise}
      */
-    var disableKeyboardPolicies = function () {
-        if (_cachedKeyboardPolicies || _cachedKeyboardMode) {
+    var disablePolicies = function () {
+        if (_cachedPolicyState) {
             return Promise.reject(new Error("Can not disable keyboard policies if a cached state exists"));
         }
 
-        var policyStore = this.flux.store("policy"),
-            kind = PolicyStore.eventKind.KEYBOARD;
-
-        _cachedKeyboardPolicies = policyStore.getPolicies(kind);
+        var policyStore = this.flux.store("policy");
+        
+        _cachedPolicyState = {};
```

Comment:
Seems like it would match existing style better and otherwise be no worse to just do this in one fell swoop with an object literal?